community-roles.md

```diff
@@ -9,22 +9,15 @@ Roles are progressive, so each include responsibilities, requirements and defini
   - [Community Member](#community-member)
   - [Project Member](#project-member)
   - [Maintainer](#maintainer)
-  - [Oversight Committee](#oversight-committee)
-- [Processes](#processes)
-  - [Inactivity](#inactivity)
-  - [Involuntary Removal](#involuntary-removal)
-  - [Stepping Down/Emeritus Process](#stepping-downemeritus-process)
-  - [Stepping Back Into a Role](#stepping-back-into-a-role)
-  - [Contact](#contact)
+  - [flux2 maintainers](#flux2-maintainers)
 
 ## Roles
 
 Most of the roles defined herein are defined by membership in a certain GitHub organization or team:
 
-* [fluxcd org](https://github.com/fluxcd): The organization under which all of Flux's activity on GitHub is captured.
-* [@fluxcd/flux2-maintainers](https://github.com/orgs/fluxcd/teams/flux2-maintainers): The team comprised of all maintainers of Flux v2 components.
-* [@fluxcd/maintainers](https://github.com/orgs/fluxcd/teams/maintainers): The team comprised of all maintainers of both, Flux v1 and Flux v2 components.
-* [@fluxcd/oversight-committee](https://github.com/orgs/fluxcd/teams/oversight-committee): The team comprised of all [Oversight Committee] members.
+- [fluxcd org](https://github.com/fluxcd): The organization under which all of Flux's activity on GitHub is captured.
+- [@fluxcd/flux2-maintainers](https://github.com/orgs/fluxcd/teams/flux2-maintainers): The team comprised of all maintainers of Flux v2 components.
+- [@fluxcd/maintainers](https://github.com/orgs/fluxcd/teams/maintainers): The team comprised of all maintainers of the various projects in the FluxCD organisation: Flux v1, Flux v2, Flagger, GitOps Toolkit, etc.
```

Comment:
Reduce possible confusion about whether we also use Git repos for non-code contributions, and whether maintainers of certain projects can focus on non-code contributions (fluxcd/community, fluxcd/website which also includes docs).

Note the community role of [Project Member](https://github.com/fluxcd/community/blob/main/community-roles.md#project-member) does already say this:

> "Project Members are [Community Members](https://github.com/fluxcd/community/blob/main/community-roles.md#community-member) who contribute directly to the project and add value to it. This can be through code, documentation, taking part in bug scrubs, etc."

But it can't hurt to reinforce this message in various places where we mention maintainers by specifically including these repos in the list of examples.

```suggestion
- [@fluxcd/maintainers](https://github.com/orgs/fluxcd/teams/maintainers): The team comprised of all maintainers of the various projects in the FluxCD organisation: Flux v1, Flux v2, Flagger, GitOps Toolkit, website and community repos, etc.
```